R/LSTdecisionTree.R

```diff
@@ -207,7 +207,6 @@ LSTdecisionTree <- function(jaspResults, dataset = NULL, options) {
   
   jaspResults[["DecisionTree"]] <- createJaspContainer("")
   jaspResults[["DecisionTree"]][["Heading"]] <- plot1
-  jaspResults[["DecisionTree"]][["Text"]] <- createJaspHtml("Some explanation and maybe a link to the test?", "p")
```

Comment:
If the idea is that there should be actual text in here at some point (just now now), I would leave this as a comment like:
```r
  # jaspResults[["DecisionTree"]][["Text"]] <- createJaspHtml("Some explanation and maybe a link to the test?", "p") # TODO: write an informative explanation.
```
The word TODO is easy to search for.

If there should never be any text here, then I would indeed remove it.

Reply:
I didn't discuss this with anyone yet, but IMO there should be some text and (if possible) a link to test or at least a description where to find it in JASP. So I'll do as you suggested!